### ISSUE TITLE
Rumble plugin does not allow login

### ISSUE THREAD

**Issue by RichardBronosky** [futo-org/grayjay-android#631]
Trying to log in to Rumble account opens an embedded web view rather than opening the default browser. I use Google sign-in for Rumble, and the Google sign in page never completes loading. Opening the sign in link in the default browser would resolve this. Does the Gray Jay authentication process allow for use of an external browser?

**Comment by flying-night-slasher** [comment]
I'm having the same problem but I'm not sure if it's the new sign in ui

**Comment by mooneyto** [comment]
Yeah, seems to be creating a lot of issues. I wish we could have the ability to open in browser, that would fix so much.
